<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Maintab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>651</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="modeBasic">
     <attribute name="title">
      <string>Mode Basic</string>
     </attribute>
     <widget class="QPushButton" name="btn_change_mode_basic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode basic</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Roll Pitch Yaw</string>
      </property>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Roll:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoxRoll">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoxPitch">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoxYaw">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Sensor Status</string>
      </property>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MPU6050:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoxMPU">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>541</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="modeIMUCalibration">
     <attribute name="title">
      <string>Mode IMU Calibration</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>631</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Gyro</string>
      </property>
      <widget class="QLineEdit" name="tb_gx">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gx:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gy:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gz:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gx offset:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gy offset:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_gy">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_gz">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_gx_offset">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_gy_offset">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gz offset:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_gz_offset">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_gyro_calib">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>20</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_stand_point">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tb_stand_point_current">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Error:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Error Offset:</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_mode_imu_write_params">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Write gyro offset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode_imu_save_params">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save params</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_change_mode_imu_calibration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode IMU Calibration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode_imu_load_params">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load params</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modePIDTunnig">
     <attribute name="title">
      <string>Mode PID Tunning</string>
     </attribute>
     <widget class="QPushButton" name="btn_change_mode_pid_tunning">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode PID tunnning</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>KP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>KI:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>KD:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode_pidt_load_params">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load params</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode_pidt_write_params">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Write params</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode_pidt_save_params">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save params</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_KP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_KI">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_KD">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_step_KP">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_step_KI">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_step_KD">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="com">
     <attribute name="title">
      <string>COM</string>
     </attribute>
     <widget class="QLineEdit" name="tbSend">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tbReceive">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>631</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_respond_ok">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Respond OK</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QComboBox" name="cbCOM">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>COM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbBaud">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCOM">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>22</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenServer">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbIPAddress">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sever address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbPort">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>9000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected from:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbConnectedIP">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
